engine/render/renderer/Core.h

```diff
@@ -13,6 +13,7 @@
 #include <cstdint>
 #include <iostream>
 
+#include "../../utils/HeapArray.h"
 #include "utils/Array.h"
 #include "utils/Hash.h"
 #include "utils/StackArray.h"
```

Comment:
utils classes should be brought in with without relative pathing, such as `#include <utils/HeapArray.h>` etc.

Reply:
Good catch.

Reply:
Changed.